README.md

```diff
@@ -55,7 +55,24 @@ Then:
 
 ## Configuration
 
-AWS credentials can be passed as environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. AWS region must be passed via `AWS_REGION`.
+AWS credentials can be passed as environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+Additional configuration can be supplied in a configuration file and might differ between collectors.
+
+An example file can look like this - anything in default (`timeout` and `regions`) will act as a fallback for the exporter specific setting if not set.
+
+```yaml
+general:
```

Comment:
as we remove `all`, i'd say we can also remove general/default?

Reply:
I had it for the fallback feature - but the more I look at the code for that the less I like it.

I'm happy doing the 'explicit is better than implicit' way and having to specify all keys for all exporters.

Reply:
I removed the default config and added the VPC and Route53 configs as examples how to extend the BaseConfig.

I guess that way should work for you, to add whatever additional configuration you want for RDS.